dvm-helper/dvm-helper.go

```diff
@@ -15,12 +15,14 @@ import "github.com/getcarina/dvm/dvm-helper/url"
 import "github.com/google/go-github/github"
 import "github.com/codegangsta/cli"
 import "github.com/kardianos/osext"
+import "golang.org/x/oauth2"
```

Comment:
This was the go-github documented way of doing it. https://github.com/google/go-github#authentication